resources/nethealth/nethealth.yaml

```diff
@@ -0,0 +1,222 @@
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: nethealth
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - nodes
+    verbs:
+      - list
+      - watch
+      - get
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: monitoring
+  name: nethealth
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: nethealth
+subjects:
+- kind: ServiceAccount
+  name: nethealth
+  namespace: monitoring
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: nethealth
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: nethealth
+subjects:
+- kind: ServiceAccount
+  name: nethealth
+  namespace: monitoring
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nethealth
+  namespace: monitoring
+  labels:
+    app: nethealth
+spec:
+  selector:
+    matchLabels:
+      k8s-app: nethealth
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        k8s-app: nethealth
+      annotations:
+        seccomp.security.alpha.kubernetes.io/pod: docker/default
+    spec:
+      terminationGracePeriodSeconds: 5
+      serviceAccountName: nethealth
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+        # Tolerate all taints
+        - effect: NoSchedule
+          operator: Exists
+        - effect: NoExecute
+          operator: Exists
+      containers:
+        - name: nethealth
+          image: quay.io/gravitational/nethealth-dev:6.0.0-3-gfb57cbfb-dirty
+          command:
+            - /nethealth
+          args:
+            - run
+          env:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          securityContext:
+            allowPrivilegeEscalation: false
+            readOnlyRootFilesystem: true
+            runAsUser: 0
+            capabilities:
+              drop:
+              - all
+              add: 
+              - NET_RAW
+          imagePullPolicy: Always
+          volumeMounts:
+            - mountPath: /tmp
+              name: tmpfs
+          ports:
+          - name: metrics
+            containerPort: 9801
+            protocol: TCP
+      volumes:
+        - name: tmpfs
+          emptyDir:
+            medium: Memory
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nethealth
+  namespace: monitoring
+  labels:
+    k8s-app: nethealth
+spec:
+  selector:
+    k8s-app: nethealth
+  ports:
+  - name: metrics 
+    protocol: TCP
+    port: 9801
+    targetPort: metrics
+---
+apiVersion: monitoring.coreos.com/v1
```

Comment:
The ServiceMonitor object is part of prometheus operator, and shouldn't be included.